README.md

```diff
@@ -41,7 +41,13 @@ root@raspberrypi:~#
 
 ##### Prerequisites
 
-[KVM](https://help.ubuntu.com/community/KVM/Installation)
+[KVM](https://help.ubuntu.com/community/KVM/Installation) and rdiff
```

Comment:
Thanks a lot for identifying the missing documentation, @necrodome ! This will definitely be useful until we automate ``KVM`` and ``rdiff`` installation during in installation scripts ([x86-64](https://github.com/myechuri/ukd/blob/master/scripts/install-x86-64.sh) and [aarch64](https://github.com/myechuri/ukd/blob/master/scripts/install-aarch64.sh)).